src/sentry/web/forms/accounts.py

```diff
@@ -483,12 +478,49 @@ def save(self):
         all_orgs = set(self.fields['organizations'].queryset.values_list('id', flat=True))
         UserOption.objects.set_value(
             user=self.user,
-            project=None,
             key='reports:disabled-organizations',
             value=list(all_orgs.difference(enabled_orgs)),
         )
 
 
+class NotificationDeploySettingsForm(forms.Form):
+    CHOICES = [
+        (UserOptionValue.all_deploys, 'All deploys'),
+        (UserOptionValue.committed_deploys_only, 'Deploys with your commits'),
```

Comment:
should this stuff be translated? not sure how we do translations outside of templates.. maybe we don't.